assets/javascripts/validation/form.js

```diff
@@ -117,15 +117,21 @@ var createErrorSummaryListItem = function ($liElement, validator, errorDetail) {
  * @param invalidInputs
  */
 var flushHiddenElementErrors = function (invalidInputs) {
-  for(var inputName in invalidInputs) {
-    var $elem = $('[name="' + inputName + '"]');
+  for (var inputName in invalidInputs) {
+    var $elem = $('[name="' + inputName + '"]'),
+      inputType = $elem.prop('type');
 
     if ($elem.is(':hidden')) {
       delete invalidInputs[inputName];
-      $elem.val('');
-      $elem.closest('.form-field-group').removeClass('form-field-group--error');
+
+      // filter out radios and checkboxes before clearing the element value   
+      if (inputType !== 'radio' && inputType !== 'checkbox') {
+        $elem.val('');
```

Comment:
Essentially the validation module shouldn't be concerned with clearing form data, only validating it